docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml

```diff
@@ -0,0 +1,155 @@
+$linkedData:
+  term: MonthlyAggregateDeliveryStatementCredential
+  '@id': https://w3id.org/traceability#MonthlyAggregateDeliveryStatementCredential
+title: Monthly Aggregate Delivery Statement Credential
+tags:
+   - Oil and Gas
+description: >-
+  Credential containing aggregate statements indicating item(s) delivered during the month
+type: object
+properties:
+  '@context':
+    type: array
+    readOnly: true
+    const:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+    default:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+    items:
+      type: string
+      enum:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    readOnly: true
+    const:
+      - VerifiableCredential
+      - MonthlyAggregateDeliveryStatementCredential
+    default:
+      - VerifiableCredential
+      - MonthlyAggregateDeliveryStatementCredential
+    items:
+      type: string
+      enum:
+        - VerifiableCredential
+        - MonthlyAggregateDeliveryStatementCredential
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    $ref: ../common/Organization.yml
+  credentialSchema:
+    type: object
+    properties:
+      id:
+        title: Id
+        description: The URL of the schema file to use to validate the shape of the JSON object
+        type: string
+        format: uri
+        example: https://w3id.org/traceability/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatementCredential.yml
+        default: https://w3id.org/traceability/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatementCredential.yml
+        readonly: true
+      type:
+        title: Type
+        description: The type of validation to be run against the defined schema
+        const: OpenApiSpecificationValidator2022
+  credentialSubject:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - MonthlyAggregateDeliveryStatementCredential
+    default:
+      - MonthlyAggregateDeliveryStatementCredential
+    items:
+      type: string
+      enum:
+        - MonthlyAggregateDeliveryStatementCredential
+    deliveryStatements:
+      type: array
+      readOnly: true
+      $ref: ../common/DeliveryStatement.yml
+  proof:
+    $ref: ../snippets/proof.yml
+additionalProperties: false
+required:
+  - '@context'
+  - type
+  - issuanceDate
+  - issuer
+  - credentialSubject
+example: |-
+  {
+    "type": [
+      "VerifiableCredential",
+      "MonthlyAggregateDeliveryStatementCredential"
+    ],
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:web:organization.example.com",
```

Comment:
```suggestion
    "id": "[suggest to change change for `https://organization.example.com/credentials/123`](https://organization.example.com/credentials/123)",
```

Reply:
the other examples look like "id": "urn:credential:1a667a1c-f761-4172-b110-5610ad25f50b"